baobab/lims/browser/viral_genomic_analyses/configure.zcml

```diff
@@ -0,0 +1,63 @@
+<configure
+        xmlns="http://namespaces.zope.org/zope"
+        xmlns:browser="http://namespaces.zope.org/browser"
+        i18n_domain="baobab.lims">
+
+    <browser:page
+        for="baobab.lims.interfaces.IViralGenomicAnalyses"
+        name="folder_view"
+        class=".viral_genomic_analyses.ViralGenomicAnalysesView"
+        permission="zope2.View"
+        layer="bika.lims.interfaces.IBikaLIMS"
+    />
+
+    <browser:page
+      for="Products.CMFPlone.interfaces.IPloneSiteRoot"
+      name="ajax_save_viral_genomic_analysis"
+      class=".ajax.viral_genomic_analysis.AjaxCreateViralGenomicAnalysis"
+      permission="zope.Public"
+    />
+
+    <browser:page
```

Comment:
@coertquinton please remove line 21 - line 26, since file 
 has been deleted baobab/lims/browser/viral_genomic_analyses/ajax/extract_genomic_material.py